graylog2-web-interface/src/components/streamrules/HumanReadableStreamRule.jsx

```diff
@@ -23,7 +20,21 @@ class HumanReadableStreamRule extends React.Component {
     return undefined;
   };
 
+  _inputName = (inputId) => {
```

Comment:
The function name is giving the wrong impression of what it's doing. It should be something like `findInput`.